todo_app/tests/client_it/test_client.py

```diff
@@ -2,58 +2,52 @@
 import pytest
 from dotenv import find_dotenv, load_dotenv
 from todo_app import app
-import requests
-
+import mongomock
 
 @pytest.fixture
 def client():
     # Use our test integration config instead of the 'real' version
     file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
 
-    # Create the new app.
-    test_app = app.create_app()
-
-    # Use the app to create a test_client that can be used in our tests.
-    with test_app.test_client() as client:
-        yield client
+    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+        # Create the new app.
+        test_app = app.create_app()
+        # Use the app to create a test_client that can be used in our tests.
+        with test_app.test_client() as client:
+            yield client
 
 def test_index_page(monkeypatch, client):
-    # This replaces any call to requests.get with our own function
-    monkeypatch.setattr(requests, 'get', stub)
+ 
+    insert_data_to_db()
 
     response = client.get('/')
-
+    
+    print(f"response {response}")
+    
     decoded_data = response.data.decode()
 
     assert response.status_code == 200
     assert 'Test card to do' in decoded_data
     assert 'Test card done' in decoded_data
-class StubResponse():
-    def __init__(self, fake_response_data, fake_status_code):
-        self.fake_response_data = fake_response_data
-        self.status_code = fake_status_code
-    def json(self):
-        return self.fake_response_data
 
+def insert_data_to_db():
```

Comment:
Here you seem to be still using the "list" response format from Trello, but that's not what our MongoDB necessarily wants, and I'm not sure you're using the right mocked DB. The easiest solution here is to take advantage of the mocking & the work you've already done to set up your DB:
```python
from todo_app.data.db_items import add_item, move_card_to_new_list

....

def insert_data_to_db():
    add_item('Test card to do')
    done_id = add_item('Test card done')
    move_card_to_new_list(done_id, "To Do", "Done")
```